app/gui2/src/components/GraphEditor/collapsing.ts

```diff
@@ -131,14 +132,27 @@ function findSafeMethodName(module: Ast.Module, baseName: string): string {
 const MODULE_NAME = 'Main'
 const COLLAPSED_FUNCTION_NAME = 'collapsed'
 
+interface CollapsingResult {
+  /** Ast edits to the module. */
+  edit: Ast.MutableModule
```

Comment:
The edit module should be received from outside, rather than being created by the collapse function. This likely also causes a conflict now, as I remember changing that once.

Reply:
Fixed